apps/condo/domains/billing/schema/BillingAccount.test.js

```diff
@@ -510,4 +522,157 @@ describe('BillingAccount', () => {
             }, `${UNEQUAL_CONTEXT_ERROR}:property:context] Context is not equal to property.context`)
         })
     })
-})
\ No newline at end of file
+
+    describe('DiscoverServiceConsumers hook', () => {
+        let context
+        beforeAll(async ()=>{
+            const { context: billingContext }  = await makeContextWithOrganizationAndIntegrationAsAdmin(
+                {}, {}, { status: CONTEXT_FINISHED_STATUS }
+            )
+            context = billingContext
+        })
+        test('should create one ServiceConsumer when BillingAccount is created', async () => {
+            const user = await makeClientWithResidentUser()
+            const unitName = faker.random.alphaNumeric(8)
+            const { address, addressMeta } = buildFakeAddressAndMeta(false)
+            const [organization] = await registerNewOrganization(admin)
```

Comment:
Be close to real-life since this tests are important, so we need sure that all accesses are enough / all actions are legit

Reply:
(Also you've already created an organization and a context in beforeAll)